Sources/PerformanceTesting/PerformanceTestCase.swift

```diff
@@ -26,7 +26,8 @@ open class PerformanceTestCase: XCTestCase {
 
     // MARK: Instance Methods
 
-    /// Benchmarks the performance of a closure.
+    /// - Returns: An array of two-tuples containing the size of the structure and the average time
```

Comment:
"size of the structure" seems too specific at this point - i feel like if we change it here then we should change the labels in the routine as well. can we keep it to `testPoint` for now?